pyproject.toml

```diff
@@ -104,6 +104,7 @@ PyYAML = "~5.4.1"
 social-auth-app-django = "^4.0.0"
 # Rendering of SVG images (for rack elevations, etc.)
 svgwrite = "~1.4.1"
+django-bootstrap3 = "^15.0.0"
```

Comment:
I don't understand why the non-admin UI is able to use Bootstrap 3 without needing a dedicated Python package, yet the admin UI needs to add a new dependency? This feels like a step backwards, especially as we look toward the future of the Nautobot UI.